dataset.json

```diff
@@ -0,0 +1,21 @@
+
+
+{
+    "titles" : [
+               "ranting about jira",
+               "siting in a meeting", 
+               "squashing bugs", 
+               "centering divs", 
+               "waiting for staging deployment",
+               "writing tests",
+               "pushing directly to prod",
+               "doing code review", 
+               "deleting jira tickets", 
+               "Creating a pull request",
+               "forgot to run DB migrations", 
+               "Broke production", 
+               "running on staging ENV", 
+               "merging git branches", 
+               "releasing the MVP", "finding JIRA tickets"
+            ]
+}
```

Comment:
Hey its a nitpick but can you format the json file?
Use the following command to do so `python3 -m json.tool <filepath>`

Reply:
Reminds me, just FYI, I noticed some missing newlines at EOL as well which can introduce hard-to-debug line ending issues. I suggest configuring `git` to do it automatically for you whenever you commit changes. See [the docs](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings) for more information on the same.

cc @SameerSahu007